device-types/Raritan/PX3-5292R-M11.yaml

```diff
@@ -0,0 +1,58 @@
+---
+manufacturer: Raritan
+model: PX3-5292R-M11
+slug: px3-5292r-m11
+part_number: PX3-5292R-M11
+u_height: 1
+is_full_depth: false
```

Comment:
Please add the `weight` and `weight_unit` data point if you have/can find it

Reply:
weight and weight_unit added